main.go

```diff
@@ -100,6 +104,11 @@ func printf(format string, a ...interface{}) (n int, err error) {
 func main() {
 	flag.Parse()
 
+	if showVersion {
+		fmt.Println("httpstat", version)
```

Comment:
We should also print the `runtime.Version()`. It may be nice to know the go runtime version we are running against.